src/game/Commands/Level3.cpp

```diff
@@ -3906,6 +3906,26 @@ bool ChatHandler::HandleGetAngleCommand(char* args)
     return true;
 }
 
+bool ChatHandler::HandleFacemeCommand(char* /*args*/)
+{
+    Unit* target = getSelectedUnit();
+    if (!target || !m_session->GetPlayer()->GetSelectionGuid())
+    {
+        SendSysMessage(LANG_SELECT_CHAR_OR_CREATURE);
+        SetSentErrorMessage(true);
+        return false;
+    }
+
+    if (target->GetTypeId() == TYPEID_PLAYER)
+    {
+        target->SetFacingToObject(m_session->GetPlayer());
+        target->CastSpell(target, 5133, false);
```

Comment:
You can use `target->InterruptNonMeleeSpells(true)` rather than triggering another cast